src/Dataflow/Serialization/Network/Importer/NetworkIO.cc

```diff
@@ -208,7 +213,102 @@ const std::map<std::string, std::string> LegacyNetworkIO::moduleRenameMap_ =
     { "CalculateFieldData2", "CalculateFieldData" },
     { "CalculateFieldData3", "CalculateFieldData" },
     { "CalculateFieldData4", "CalculateFieldData" },
-    { "CalculateFieldData5", "CalculateFieldData" }
+    { "CalculateFieldData5", "CalculateFieldData" },
```

Comment:
@jessdtate In this file, could you examine this list of unconverted modules and see if any stick out for conversion, other than the ones we know we have to work on soon.

Reply:
modules that I would add to the next level module conversion:

CalculateNodeNormals
VectorMagnitude
ApplyFilterToFieldData
ShowMatrix (or something like it)
ReportScalarFieldStats
GetDomainStructure